src/components/Photos.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Photo from './Photo';
+// import './Photos.css';
+
+function Photos({ urls }) {
+  const listOfUrls = urls.map(url => {
+    return (
+      <li key={url}>
+        <Photo url={url}/>
+      </li>
+    );
+  });
+
+  return (
+    <ul>
+      <li>{listOfUrls}</li>
```

Comment:
```suggestion
      {listOfUrls}
```
Your "li" tags are already being rendered by "listOfUrls" on line 7. No need to repeat them. Browser notifies you of this with:
> Warning: validateDOMNesting(...): ```<li>``` cannot appear as a descendant of ```<li>```...